Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:22.10
+
+ENV TZ=Europe/Berlin
+# Installing python and pip on user level so we need to add it to the PATH
+ENV PATH="/home/mrbeam/.local/bin:/home/mrbeam/bin:${PATH}"
+
+SHELL ["/bin/bash", "-c"]
+
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN mkdir /home/mrbeam/ \
+    && groupadd -g 999 mrbeam \
```

Comment:
Should we change the user to pi, so it will be the same as on our image?

Reply:
Hmm... Good question... Why is it called pi? If we call it "pi" here it will be confusing, I think. This is meant to be run anywhere not in a raspberry... 🤔

Reply:
It will work with mrbeam too, was just an idea to have the same user. But it is fine for me with the mrbeam user too